dbcfeederlib/serverclientwrapper.py

```diff
@@ -65,7 +65,7 @@ def start(self):
 
         self._kuksa = KuksaClientThread(self._client_config)
         self._kuksa.start()
-        self._kuksa.authorize()
+        self._kuksa.authorize(token_or_tokenfile=self._token_path)
```

Comment:
Previously we did not consider what token was given in the config file, it worked anyway as kuksa-python-sdk had a default one which was the same one as we used in tests. This is an example on what we need to do to update client to latest kuksa-client. It should only affect VISS/websocket, but I assume we want to make sure thatw e support Websocket at least until end of year when Kuksa Server reaches end of life